src/test/ui/consts/const-eval/ub-wide-ptr.rs

```diff
@@ -149,7 +149,7 @@ const TRAIT_OBJ_CONTENT_INVALID: &dyn Trait = &unsafe { BoolTransmute { val: 3 }
 //~^ ERROR it is undefined behavior to use this value
 
 // # raw trait object
-const RAW_TRAIT_OBJ_VTABLE_NULL: *const dyn Trait = unsafe { DynTransmute { bad: BadDynRepr { ptr: &92, vtable: 0 } }.rust};
+const RAW_TRAIT_OBJ_VTABLE_NULL: *const dyn Trait = unsafe { DynTransmute { bad: BadDynRepr { ptr: &92, vtable: 0 } }.raw_rust};
```

Comment:
what happens when the original code is used? Maybe test both?

Reply:
Added `ub-wide-ptr-2.rs` with the old version of this test.

Reply:
Any reason to have a separate file for that? We often combine many tests in one file here, both to make it easier to see "all tests related to wide ptrs" and to reduce the overall number of tests (and thus the time it takes to run all of them).